qemu/tests/cfg/vsock.cfg

```diff
@@ -2,6 +2,8 @@
     no Windows
     type = vsock_test
     vsocks = vhost_vsock0
+    s390x:
+        virtio_dev_iommu_platform = on
```

Comment:
If s390x can launch a guest vsock without `virtio_dev_iommu_platform = on`, make it consistent with the behavior of other platforms.

Reply:
removed